selenium/codenvy-selenium-test/src/test/java/com/codenvy/selenium/dashboard/onprem_organization/BulkOnpremOrganizationDeletionTest.java

```diff
@@ -58,7 +59,7 @@
 
   @BeforeClass
   public void setUp() throws Exception {
-    dashboard.open();
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
```

Comment:
If we need to add almost complete duplicates of organization tests of Che only for usage of `dashboard.open(adminTestUser.getName(), adminTestUser.getPassword())` I would prefer to use `dashboard.open(String userName, String userPassword)` method in Che organization tests with nullable paremeters, and in case when they = null we can use default user credentials instead.